### PR TITLE
Merge changes from `22.08` into `22.10`

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR merges the changes from `branch-22.08` into `branch-22.10`.

### CODE REVIEW

**Review by jakirkham — APPROVED** [review]

File: recipes/xgboost/meta.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {% set xgboost_repo = environ.get('XGBOOST_REPO', 'dmlc/xgboost') %}
 {% set xgboost_branch = environ.get('XGBOOST_BRANCH', 'main') %}
 {% set build_number = environ.get('XGBOOST_BUILD_NUMBER', 1) %}
-{% set rapids_version = environ.get('RAPIDS_VER', '0.16') %}
+{% set rapids_version = environ.get('RAPIDS_VER', '22.06') %}
```
Comment (dantegd) [inline recipes/xgboost/meta.yaml:9]: shouldn't this be 22.10?

Comment (ajschmidt8) [reply recipes/xgboost/meta.yaml]: Done in 0f9ef2f and c00033a

Comment (ajschmidt8) [reply recipes/xgboost/meta.yaml]: FWIW, the `RAPIDS_VER` environment variable is set in CI. So the value we changed here is just a fallback value.

Comment (jakirkham) [reply recipes/xgboost/meta.yaml]: Thanks AJ! 🙏

**Pull request merged**